Tests/ApplicativeRouterHttpPipelineSupportTests/ApplicativeRouterHttpPipelineSupportTests.swift

```diff
@@ -14,23 +14,21 @@ class ApplicativeRouterHttpPipelineSupportTests: XCTestCase {
 
     let middleware: Middleware<StatusLineOpen, ResponseEnded, Prelude.Unit, Data?> =
       route(router: router)
-        <| (
-          writeStatus(.ok)
-            >>> { $0 |> respond(text: "Recognized route: \($0.data)") }
-    )
+        <| writeStatus(.ok)
+        >-> { $0 |> respond(text: "Recognized route: \($0.data)") }
 
     assertSnapshot(
-      matching: middleware(connection(from: URLRequest(url: URL(string: "/")!))),
+      matching: middleware(connection(from: URLRequest(url: URL(string: "/")!))).perform(),
```

Comment:
I wish I didn't need to do `.perform()` and instead could do `>>> perform` at the end of the middleware declaration above, _BUT_ swift gets confused with the `.perform` on `NSObject` since we are in an `XCTest` :/